dali/operators/image/CMakeLists.txt

```diff
@@ -19,11 +19,14 @@ add_subdirectory(crop)
 
 add_subdirectory(convolution)
 add_subdirectory(distortion)
-add_subdirectory(resize)
+if ($BUILD_CVCUDA)
+    add_subdirectory(filter)
+endif()
```

Comment:
Do we have that or should it be marked as a spoiler alert for the next PR?

Reply:
Spoiler alert ;)
Removed